apps/cms-v2/src/logger.ts

```diff
@@ -1,23 +1,14 @@
-import { Logger, ISettingsParam, ILogObj } from "tslog";
+import { logger, createLogger } from "@saleor/apps-logger";
 
-/*
- * TODO: Add loggerContext
- * TODO: Migrate to shared
- */
-export const logger = new Logger<ILogObj>({
-  argumentsArrayName: "payload",
-  minLevel: parseInt(process.env.NEW_APP_LOG_LEVEL ?? "3", 10),
-});
+logger.settings.maskValuesOfKeys = ["metadata", "username", "password", "apiKey"];
```

Comment:
question: If this logger is imported from a shared package, won't that cause interference that we change a global value?

Reply:
each app is built separately so this code is never executed for other apps